include/exadg/acoustic_conservation_equations/user_interface/numbers.h

```diff
@@ -0,0 +1,37 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2023 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef EXADG_ACOUSTIC_CONSERVATION_EQUATIONS_USER_INTERFACE_NUMBERS_H_
+#define EXADG_ACOUSTIC_CONSERVATION_EQUATIONS_USER_INTERFACE_NUMBERS_H_
+
+namespace ExaDG
+{
+namespace Acoustics
+{
+namespace block
+{
+unsigned int const pressure = 0;
+unsigned int const velocity = 1;
+} // namespace block
```

Comment:
Why did you make this information public? Isn't it the responsibility of the spatial operator, which also is responsible for the dof-handlers, initialization of dof-vectors, etc.

Also, it is not clear to me why this is part of the user-interface. In terms of the user interface (e.g. creating new acoustics applications), to which extent is it relevant to know these indices? My goal would be to make this a detail of the internal implementation.

The postprocessor might need this information. But there we already have `using AcousticsOperator = SpatialOperator<dim, Number>;`, so one could write `AcousticOperator::block_index_pressure` in the postprocessor.

Reply:
I restructured accordingly [8b5e62c](https://github.com/exadg/exadg/pull/592/commits/8b5e62cf1ed3e467bd025f27a38c05fc45dfb543)